adobe_vipm/flows/fulfillment/transfer.py

```diff
@@ -1249,6 +1249,7 @@ def __call__(self, client, context, next_step):
         """Get Adobe customer and saves it to the context."""
         adobe_client = get_adobe_client()
         context.customer_id = context.adobe_transfer_order["customerId"]
+        context.adobe_customer_id = context.adobe_transfer_order["customerId"]
```

Comment:
context.customer_id and context.adobe_customer_id contain the same value -> context.adobe_transfer_order["customerId"], do we really need both? 
I think we can remove context.customer_id and keep the adobe_customer_id to be more explicit. 

I was dealing with a similar case here -> https://github.com/softwareone-platform/swo-adobe-vipm-extension/pull/682